samples/net/sockets/http_server/Kconfig

```diff
@@ -38,4 +38,16 @@ config NET_SAMPLE_CERTS_WITH_SC
 	  Enable this flag, if you are interested to run this
 	  application with signed certificates and keys.
 
+config NET_SAMPLE_WEBSOCKET_SERVICE
```

Comment:
please remember to also update the README to mention WebSocket: at a minimum, this new option should be mentioned in "Server Customization" section, but also would be nice to have some basic instructions on how to "play" with the websocket.

BTW, but I know that you have it on your list already to polish the documentation, the README should likely also indicate the various endpoints and associated methods that are exposed by the server, e.g. `/dynamic` would be worth mentioning/explaining.